### DISCUSSION TITLE
[Question/Help]  play music , bases on track number , not based on alphatic order?

### DISCUSSION THREAD

**Discussion by OmiChrome** [hooke007/mpv_PlayKit#266] [z_ENG_discuss]
mpv defaults  play track based on alphabetic order , but based on track ID , or track number 

this thread is a _repost _ [reddit-question ](https://www.reddit.com/r/mpv/comments/10sp0ge/questionhelp_force_to_play_track_based_upon_track/) by me 
please help

**Comment by hooke007** [comment]
<s>As far as I know mpv doesn't support reading music-files' tags.</s>
https://mpv.io/manual/master/#command-interface-metadata
It could be handled by external script.

**Reply by OmiChrome** [reply]
what can i do then ?

**Reply by hooke007** [reply]
Write a script.

**Reply by OmiChrome** [reply]
im not a programer

**Reply by hooke007** [reply]
Other possible solution:
- To find and wait for someone to write it.
- Use a music player instead of video player.
- Manually create a m3u playlist for each album.
- Manually convert those files into one iso file.

**Reply by OmiChrome** [reply]
Whats the matter of `iso` file? i dont get it

**Reply by hooke007** [reply]
Audio CD format. Sry I have no interest on it, you have to DIY.

**Reply by OmiChrome** [reply]
my collection is over 48GB , which is a lot , also im running out of storage , i can't think of making separate `iso` file, 
even when i have no idea of , , what iso can help me in this ? 
do iso makes mpv read tag different ly?

**Reply by hooke007** [reply]
> Sry I have no interest on it, you have to DIY.
